<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewPlayer</class>
 <widget class="QWidget" name="PreviewPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PreviewPlayer</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="playButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pauseButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="timeLine">
        <property name="styleSheet">
         <string notr="true">QSlider{
background-color:rgb(104, 104, 104)
}
QSlider::groove:horizontal {
height: 5px;
border: 0px solid;
margin: 2px 0;
}
QSlider::handle:horizontal {
background: orange;
height: 5px;
width: 8px;
margin: -10px 0;
border-radius: 18px;
}l</string>
        </property>
        <property name="maximum">
         <number>15000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="styleSheet">
         <string notr="true">QSlider{
background-color: &quot;white&quot;;
}</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="sliderPosition">
         <number>70</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="previewLabel">
        <property name="styleSheet">
         <string notr="true">QLabel{
background-color:&quot;black&quot;;
border-style: solid;
    border-width: 5px;
    border-color: rgb(63, 63, 70)
}</string>
        </property>
        <property name="text">
         <string>Video</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>PreviewPlayer</receiver>
   <slot>onPlayButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>PreviewPlayer</receiver>
   <slot>onPauseButtonCLicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>PreviewPlayer</receiver>
   <slot>onStopButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPlayButtonClicked()</slot>
  <slot>onStopButtonClicked()</slot>
  <slot>onPauseButtonCLicked()</slot>
 </slots>
</ui>
